deployment/data-streams/changeset/reward-manager/general/call_set_reward_recipients_test.go

```diff
@@ -28,39 +33,67 @@ func TestCallSetRewardRecipients(t *testing.T) {
 			Weight: 500000000000000000,
 		},
 	}
-	var poolID [32]byte
-	copy(poolID[:], []byte("poolId"))
 
-	e, err := commonChangesets.Apply(t, e, nil,
+	var timelocks map[uint64]*proposalutils.TimelockExecutionContracts
+	if useMCMS {
+		e, _, timelocks = testutil.DeployMCMS(t, e, map[uint64][]common.Address{
+			chainSelector: {rewardManagerAddr},
+		})
+	}
+
```

Comment:
can we add some check somewhere (maybe in DeployMCMS?) just for our own safety - to confirm ownership transfer is completed to timelock? I see that happens within the `Apply` function. Would this test fail in anyway if that AcceptOwnership proposal was not executed?

Reply:
I believe TransferToMCMSWithTimelock is already tested, so it feels unnecessary. And yeah it would fail for sure because all these methods have an onlyOwner modifier.

Reply:
Sounds good